notebooks/en/fine_tuning_llm_to_generate_persian_product_catalogs_in_json_format.ipynb

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
<p>Suggestion: *Model Training</p><p> </p><p>Also would be nice to walk people through what you're doing when you load bnb config, lora config and more (we try to assume the reader isn't very knowledgeable)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>